arch/x86/deflate_quick.c

```diff
@@ -234,7 +215,7 @@ ZLIB_INTERNAL block_state deflate_quick(deflate_state *s, int flush) {
         }
 
         if (s->lookahead >= MIN_MATCH) {
-            hash_head = quick_insert_string(s, s->strstart);
+            hash_head = functable.quick_insert_string(s, s->strstart);
```

Comment:
Since deflate_quick itself depends on SSE, I think we can safely use quick_insert_string_sse directly here, and avoid the functable indirection. At least until we get a better insert_string or deflate_quick gets ported to another arch.

Reply:
Ok. I think the only thing preventing it from getting ported is the `compare258` function.